pytket/phir/phirgen.py

```diff
@@ -356,15 +357,26 @@ def convert_classicalevalop(op: tk.ClassicalEvalOp, cmd: tk.Command) -> JsonDict
 
 def multi_bit_condition(args: "list[UnitID]", value: int) -> JsonDict:
```

Comment:
I have a feeling this will error out if `len(args)` is 0 or 1. While these may be rare edge cases it would be good to handle them or at least raise an informative exception.

Reply:
This would make sense, but PECOS does not seem to be able to deal with `"cop": "&"` with fewer than two arguments.
```
ValueError: not enough values to unpack (expected 2, got 1)
```

Is there a way to add a "True" constant to an expression in PHIR? The AND with a single element `args == [x]` could be implemented as `x & x`, but in the case of `args == []` you'd need to be able to output True somehow.

Not sure if we'd want to do this, though. Is it OK for pytket-phir to create valid PHIR when the conditions the user inputted would not be accepted by PECOS if done directly? It's true that in this case, the action of AND on 1 and 0 arguments is not ambiguous, but it still makes me uneasy.

Reply:
The function is only called at one place when len(args) is not 1, but for generality (and reusability), it's certainly worth doing an edge case check. Implemented and pushed.